src/indigo/less/inline-edit-text-input.less

```diff
@@ -0,0 +1,77 @@
+.inline-edit-text-input {
+  display: inline;
+  position: relative;
+
+  &-form {
+    display: inline;
+  }
+
+  &-buttons {
+    position: absolute;
+    right: 10px;
+    top: -3px;
+
+    .fa-spin {
+      position: absolute;
+      top: 5px;
+      right: -35px;
+      background: #fff;
+    }
+  }
+
+  &-cancel {
+    padding: 0 5px 0 5px;
+    font-size: 120%;
+    color: #e1e1e9;
+
+    &:hover,
+    &:focus,
+    &:active {
+      text-decoration: none;
+      color: darken(#e1e1e9, 20%);
+    }
+
+    .kbc-icon-cross {
+      position: relative;
+      top: 2px;
+    }
+  }
+
+  &-submit {
+    padding: 5px 10px;
+    font-size: 60%;
+    border-radius: 15px;
+    [class^="kbc-icon-"] {
+      font-size: 200%;
+    }
+  }
+
+  &-link {
+    padding: 6px 15px 9px;
+    border-radius: 1em;
+    background-color: inherit;
+    cursor: pointer;
+
+    &:hover,
+    &:focus,
+    &:active {
+      background-color: #e9faff;
+
+      .kbc-icon-pencil {
+        opacity: 1;
+      }
+    }
+
+    .kbc-icon-pencil {
+      margin-left: 10px;
+      opacity: 0.3;
+    }
+  }
+
+  .form-control {
```

Comment:
Ja jsem pro to aby jsme nepouzivali tridy zavisly na parent komponente. 
To pak vede k zaludnym zavislostem.  Ja to smeruju na to aby kazdy element mel explicitne vlastni tridu, pak bude stylovane pekne plochy.
Pocet classnames pro html element je naopak neco, cim se nemusi setrit.

Ty ikony vyse neres, ale jestli muzes tohle oclassovat podobne jako elementy vyse, bude to cistejsi.

Pokud mas nejaky zkusenosti s psanim css systemu, budu rad kdyz se podelis.